bin/behat-prepare.sh

```diff
@@ -22,125 +23,191 @@ fi
 ###
 # Create a new environment for this particular test run.
 ###
-terminus env:create $TERMINUS_SITE.dev $TERMINUS_ENV
-terminus env:wipe $SITE_ENV --yes
+terminus env:create "$TERMINUS_SITE".dev "$TERMINUS_ENV"
+terminus env:wipe "$SITE_ENV" --yes
 
 ###
 # Get all necessary environment details.
 ###
-PANTHEON_GIT_URL=$(terminus connection:info $SITE_ENV --field=git_url)
+PANTHEON_GIT_URL=$(terminus connection:info "$SITE_ENV" --field=git_url)
 PANTHEON_SITE_URL="$TERMINUS_ENV-$TERMINUS_SITE.pantheonsite.io"
 PREPARE_DIR="/tmp/$TERMINUS_ENV-$TERMINUS_SITE"
 BASH_DIR="$( cd -P "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 ###
 # Switch to git mode for pushing the files up
 ###
-terminus connection:set $SITE_ENV git
-rm -rf $PREPARE_DIR
-git clone -b $TERMINUS_ENV $PANTHEON_GIT_URL $PREPARE_DIR
+terminus connection:set "$SITE_ENV" git
+rm -rf "$PREPARE_DIR"
+git clone -b "$TERMINUS_ENV" "$PANTHEON_GIT_URL" "$PREPARE_DIR"
 
 ###
 # Add WP Native PHP Sessions and child theme to environment
 ###
 echo "Creating a child theme called $TERMINUS_SITE"
-rm -rf $PREPARE_DIR/wp-content/themes/$TERMINUS_SITE
+rm -rf "$PREPARE_DIR"/wp-content/themes/"$TERMINUS_SITE"
```

Comment:
I see all of these quotes throughout this file were added to appease shellcheck. A better syntax would be quoting the entire string once and bracketing the variables, a lot more readable, but I care 1/10
```suggestion
rm -rf "${PREPARE_DIR}/wp-content/themes/${TERMINUS_SITE}"
```